src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27' Retina 5K"
+        class="card__img"
+      >
+      <div class="card__item-name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="card__item-code">
+        Код товара: 195434
+      </div>
+      <div class="card__item-rating">
+        <div class="card__item-rating__stars">
+          <div class="stars"></div>
+          <div class="stars"></div>
+          <div class="stars"></div>
+          <div class="stars"></div>
+          <div class="stars"></div>
+        </div>
+        <div class="card__item-rating__rewiews">
+          Отзывов: 5
+        </div>
+      </div>
+      <div class="card__price">
+        <div class="card__price-title">
+          Цена:
+        </div>
+        <div class="card__price-value">
+          69 999 грн
+        </div>
+      </div>
+      <a
+        href="#"
+        data-qa="hover"
+        class="card__purchase-button"
+      >
+        КУПИТЬ
```

Comment:
```suggestion
        Купить
```

Сделай верхний регистр через css

Reply:
done